app/parse.py

```diff
@@ -8,8 +14,55 @@ class Quote:
     tags: list[str]
 
 
+def parse_single_quote(quote_soup: BeautifulSoup) -> Quote:
+    text = quote_soup.select_one(".text").get_text(strip=True)
+    author = quote_soup.select_one(".author").get_text(strip=True)
+    tags = [tag.get_text(strip=True) for tag in quote_soup.select(".tag")]
+
+    return Quote(text=text, author=author, tags=tags)
+
+
+def parse_quotes_page() -> [Quote]:
+    page_num = 1
+    page = requests.get(BASE_URL).content
+    soup = BeautifulSoup(page, "html.parser")
+    quotes = soup.select(".quote")
+    print(f"Page {page_num}")
+
+    while soup.select_one(".next"):
+        page = requests.get(
+            BASE_URL + soup.select_one(".next > a")["href"]
+        ).content
+        soup = BeautifulSoup(page, "html.parser")
+        quotes.extend(soup.select(".quote"))
+
+        if soup.select_one(".next > a"):
```

Comment:
The check for soup.select_one(".next > a") inside the loop is redundant because the loop already depends on soup.select_one(".next") being truthy.